content/learn/case-studies/transunion.md

```diff
@@ -0,0 +1,50 @@
+---
+$order: 26
+$title: TransUnion
+$date: Mar 12, 2018
+$category: ecommerce
+$parent: /content/learn/case-studies.html
+class: case-study post-detail
+
+components:
+  - social-share
+
+headline: TransUnion scores more conversions with faster page loads via AMP
+results:
+    - stat: 3%
+      description: increase in conversion
+    - stat: 26% 
+      description: lower bounce rate
+    - stat: 2.5x
+      description: time on site
+
+logo:
+  src: "case-studies/transunion_logo.png"
+  width: 230
+  height: 65
+download: "case-studies/transunion.pdf"
+
+---
+
+
+<div class="img-left">
+    <amp-img width="400" height="780" layout="responsive" src="/static/img/case-studies/transunion_1.png"></amp-img>
+</div>
+
+Smartphones have empowered people to take control of their personal finances in a way that’s never been possible before. Important information, like credit scores, is right at our fingertips—ready to be accessed anytime, anywhere. In fact, 56% of searches for credit reporting now come from mobile devices. This affects not only consumers, but financial institutions too. TransUnion, one of the three national credit bureaus, has seen a significant increase in mobile traffic, which now accounts for the majority of visitors to the consumer services division of its site.
+
+TransUnion had a hypothesis that slow loading mobile pages could be leading to higher bounce rates and lower mobile conversion rates compared to desktop. They wanted to create a best-in-class mobile site on their CMS, HP TeamSite, to both improve the experience for users and get the highest return on their advertising investment.
```

Comment:
I noticed the PDF had a "Challenge" heading, did you purposely omit it here?

Reply:
Yes, it didn't look great on the web and it's not our standard format. I'm open to removing in the PDF if that helps ensure it matches.

Reply:
It's fine.  Just checking.